src/pybito.cpp

```diff
@@ -625,7 +643,17 @@ PYBIND11_MODULE(bito, m) {
            "Estimate the SBN parameters based on current branch lengths.")
       .def("estimate_branch_lengths", &GPInstance::EstimateBranchLengths,
            "Estimate branch lengths for the GPInstance.", py::arg("tol"),
-           py::arg("max_iter"), py::arg("quiet") = false)
+           py::arg("max_iter"), py::arg("quiet") = false,
+           py::arg("track_intermediate_iterations") = false)
+      .def("get_perpcsp_llh_surface", &GPInstance::GetPerGPCSPLogLikelihoodSurfaces,
+           "Scan the likelihood surface for the pcsps in the GPInstance.",
+           py::arg("steps"), py::arg("scale_min"), py::arg("scale_max"))
+      .def("perturb_and_track_optimization_values",
+           &GPInstance::PerturbAndTrackValuesFromOptimization,
+           "Reinitiate optimization, perturbing one branch length at a time,  and "
+           "track branch length and per pcsp likelihoods/")
```

Comment:
Does trailing slash indicate something is missing?

Reply:
Nope, fixed.